src/build/stable/commands/compile/wasm_binary/manipulate.ts

```diff
@@ -137,6 +137,16 @@ export function addCanisterMethodProxies(
             false
         );
     }
+
+    if (methodMeta?.on_low_wasm_memory !== undefined) {
```

Comment:
One question I have is if on_low_wasm_memory will automatically work in experimental mode. And do you have experimental mode tests?